### ISSUE TITLE
Unicode support

### ISSUE THREAD

**Issue by thomedw** [FDVSolutions/DynamicJasper-core-fonts#2]
Hi,

These fonts work great with english character. Unicode / CJK support will be a nice addition.

Thanks,
Thomas

**Comment by juanalvarezg** [comment]
Can you provide the fonts?

## 

_FDVSOLUTIONS | Collective Innovation_

_Juan Manuel Alvarez Gimenez_
CTO
Office:  +54 11 5239 9899 | Cel: +54 9 11 4400 0852
Connect me on: LINKEDIN http://ar.linkedin.com/in/jmalvarez | SKYPE -
juan.manuel.alvarez.gimenez
Avenida Belgrano 1580 2º piso C1093AAA , Buenos Aires, Argentina.

2016-06-13 1:51 GMT-03:00 Thomas Edwin Santosa notifications@github.com:

**Comment by thomedw** [comment]
Yes, but I don't think it's legal. I have extracted one from OS X. 

Salam,
Thomas Edwin Santosa

Pada 28 Jun 2016, pukul 20.52, Juan Manuel Alvarez Gimenez notifications@github.com menulis:

**Comment by juanalvarezg** [comment]
Hello again, thinking twice, setting the right encoding should be all you
need. People opening docs with such encoding should have proper fonts in
they machines.
The fonts provided are mostly used when in a server with no fonts at all
and you want to embed the fonts in a PDF.

## 

_FDVSOLUTIONS | Collective Innovation_

_Juan Manuel Alvarez Gimenez_
CTO
Office:  +54 11 5239 9899 | Cel: +54 9 11 4400 0852
Connect me on: LINKEDIN http://ar.linkedin.com/in/jmalvarez | SKYPE -
juan.manuel.alvarez.gimenez
Avenida Belgrano 1580 2º piso C1093AAA , Buenos Aires, Argentina.

2016-06-28 12:14 GMT-03:00 Thomas Edwin Santosa notifications@github.com:

**Comment by juanalvarezg** [comment]
Hello again, please check this example that uses Polish characters, setting
the right encoding is what really matters

http://pastebin.com/4NUgXTTA

## 

_FDVSOLUTIONS | Collective Innovation_

_Juan Manuel Alvarez Gimenez_
CTO
Office:  +54 11 5239 9899 | Cel: +54 9 11 4400 0852
Connect me on: LINKEDIN http://ar.linkedin.com/in/jmalvarez | SKYPE -
juan.manuel.alvarez.gimenez
Avenida Belgrano 1580 2º piso C1093AAA , Buenos Aires, Argentina.

2016-07-05 12:42 GMT-03:00 Juan Manuel Alvarez Gimenez <
juan.alvarez@fdvsolutions.com>:
